locksmithctl/daemon.go

```diff
@@ -352,5 +372,14 @@ func runDaemon() int {
 	close(stop)
 	wg.Wait()
 
+	if window != nil && !window.In(time.Now()) {
```

Comment:
This logic might be too simple. If you are using etcd-lock with the update window, should it grab the lock outside of the update window?